pkg/deploy/service_test.go

```diff
@@ -287,6 +288,11 @@ func (suite *ServiceTestSuite) TestCreateDeploymentAnalysisNoDefault() {
 	suite.Errorf(err, "analysis configuration block is present but default or explicit account is not set")
 }
 
+func TestBuildStrategy(t *testing.T) {
```

Comment:
sorry for this BS test...had to add because Sonarqube was mad that I altered the error message string when updating `blue-green` to `blueGreen`